include/net/sntp.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SNTP_H
+#define __SNTP_H
+
+#include <net/net_app.h>
+
+struct sntp_ctx;
+
+/**
+ * @typedef sntp_resp_cb_t
+ * @brief SNTP response callback.
+ *
+ * @details The callback is called after a sntp response is received
+ *
+ * @param ctx Address of sntp context.
+ * @param status Error code of sntp response.
+ * @param epoch_time Seconds since 1 January 1970.
+ * @param user_data The user data given in sntp_request().
+ */
+
+typedef void (*sntp_resp_cb_t)(struct sntp_ctx *ctx,
+			       int status,
+			       u64_t epoch_time,
+			       void *user_data);
+
+struct sntp_ctx {
```

Comment:
Please document the variables in this struct. Although the variables are quite obvious, we try to do this in all the header files if possible.

Reply:
OK. I will add the document for the variables.

Reply:
OK. I will add the document for the variables.